src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/VariantOverlapAnnotator.java

```diff
@@ -156,26 +158,39 @@ public VariantContext annotateOverlap(final List<VariantContext> overlapTestVCs,
     private static String getRsID(final List<VariantContext> rsIDSourceVCs, final VariantContext vcToAnnotate) {
         Utils.nonNull(rsIDSourceVCs, "rsIDSourceVCs cannot be null");
         Utils.nonNull(vcToAnnotate, "vcToAnnotate cannot be null");
+        final List<String> rsids = new ArrayList<>();
```

Comment:
Update the @return line of the comments.

Reply:
done